pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -859,6 +859,7 @@ func testMySQLProtocolClassification(t *testing.T, tr *Tracer, clientHost, targe
 }
 
 func testMySQLProtocolClassificationTLS(t *testing.T, tr *Tracer, clientHost, targetHost, serverHost string) {
+	t.Skip("MySQL+TLS classification is flaky")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**

```suggestion
    // MySQL+TLS classification is flaky
	flake.Mark(t)
```
with a 
```
import (
  "github.com/DataDog/datadog-agent/pkg/util/testutil/flake"
)
```